airbyte-integrations/connectors/source-oracle-v2/src/cdk/main/kotlin/io/airbyte/cdk/command/ConnectorCommandLinePropertySource.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.command
+
+import io.airbyte.cdk.integrations.base.JavaBaseConstants
+import io.airbyte.cdk.operation.CONNECTOR_OPERATION
+import io.airbyte.cdk.operation.OperationType
+import io.github.oshai.kotlinlogging.KotlinLogging
+import io.micronaut.context.env.MapPropertySource
+import io.micronaut.core.cli.CommandLine
+import java.io.File
+import java.nio.file.Path
+
+private val log = KotlinLogging.logger {}
```

Comment:
That reminds me of a question I wanted to run by you: What logging framework do we want to settle on? Is this the one?

Reply:
it's slf4j under the hood, and it's nice to use, so, yes please